ambry-store/src/test/java/com.github.ambry.store/IndexSegmentTest.java

```diff
@@ -515,4 +602,17 @@ private void verifyJournal(NavigableMap<MockId, IndexValue> referenceIndex, Offs
       assertFalse("One of the extraOffsetsToCheck was not found", extraEntriesCheckState.values().contains(false));
     }
   }
+
+  private class MockIndexSegmentV0 extends IndexSegment {
```

Comment:
What is this class doing that `IndexSegment` could not have? I would have thought it would be used to override the version in `IndexSegment`. Does this actually do anything in V0 other than export the value size provided?